django.mk

```diff
@@ -22,7 +22,7 @@ INTERFACE ?= localhost
 RUNSERVER_PORT ?= 8000
 PY_DIRS ?= $(APP)
 
-jenkins: check flake8 test jshint jscs
+jenkins: check flake8 test jshint eslint
```

Comment:
When introducing eslint you can also remove jshint. since both of these tools address semantic problems in javascript, we only need one.